content/en/docs/tasks/administer-cluster/coredns.md

```diff
@@ -16,16 +16,29 @@ This page describes the CoreDNS upgrade process and how to install kube-dns inst
 
 {{% capture steps %}}
 
-## Installing kube-dns with kubeadm
+## About CoreDNS
 
-In Kubernetes 1.11, [CoreDNS](https://coredns.io) has graduated to General Availability (GA)
+[CoreDNS](https://coredns.io) is a flexible, extensible authoritative DNS server that can run on a Kubernetes cluster.
+The CoreDNS project is hosted, like Kubernetes, by the [CNCF.](http://www.cncf.io)
+
+You can use CoreDNS instead of kube-dns in your cluster by directly replacing kube-dns in an existing
+deployment, or using tools like kubeadm that will deploy and upgrade the cluster for you.
+
+## Installing CoreDNS
+
+For manual deployment or replacement of kube-dns, refer to the documentation at the
+[CoreDNS GitHub project.](https://github.com/coredns/deployment/tree/master/kubernetes)
+
+## Installing CoreDNS with kubeadm
+
+In Kubernetes 1.11, CoreDNS has graduated to General Availability (GA)
 and is installed by default. To install kube-dns instead, set the `CoreDNS` feature gate
 value to `false`:
 ```
 kubeadm init --feature-gates=CoreDNS=false
 ```
```

Comment:
Here the way it is presented in the website, one can think the command-line operation is to install CoreDNS whereas it is the opposite. So I propose to detach the 2 sentences.

See below:


In Kubernetes 1.11, CoreDNS has graduated to General Availability (GA)
 and is installed by default

In order to NOT install CoreDNS but rather install kube-dns as the cluster's DNS Server, set the `CoreDNS` feature gate value to `false`:
 ```
 kubeadm init --feature-gates=CoreDNS=false
 ```

Reply:
Sorry, @fturib -- I see your concern, but the original as @neolit123 has written/edited is completely clear. We don't need the additional words -- they only get in the way.

Reply:
ok.

Reply:
see inline conversation; keep the original